apps/evm/src/clients/subgraph/queries/governanceBsc/getBscProposals/getBscProposals.graphql

```diff
@@ -0,0 +1,9 @@
+query Proposals($limit: Int = 10, $skip: Int = 0, $where: Proposal_filter) {
+  proposals(first: $limit, skip: $skip, where: $where, orderBy: startBlock, orderDirection: desc) {
+    ...ProposalFragment
+  }
+
+  total: proposals(first: 1000, where: $where) {
+    id
+  }
```

Comment:
This is related to what I mentioned in a DSU, wouldn't we have to revisit this number in a not so distant future? Ideally we'd receive the total count directly from the response

Reply:
Sorry I missed this DSU. I also think it would be best to get the total count from the response directly, but when I wrote this query for the first time and asked about this subject I was told it is not doable.
A solution for now would be to increase the "first" parameter's value to 10000, which should cover us for a long time.

Reply:
Corey pointed out we can simply remove the `first` parameter to fetch all proposals